src/index.js

```diff
@@ -17,6 +17,22 @@ let reduxStore = {
     }
   },
   entities: {
+    countries: {
+      '': {title: 'Select a country'},
```

Comment:
Noted. will fix when country data will be loaded async as well (currently it is mocked)